app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, \
```

Comment:
one test fails, I guess because on this method. I am not sure about the reason but in my case pycharm has added additional " symbols on the end of the line when put text on the new line
        return f"{{Name: {self.name}, " \
                f"Health: {self.health}, " \
                f"Hidden: {self.hidden}}}"